### ISSUE TITLE
devicon missing in NerdTREE when moving up a directory

### ISSUE THREAD

**Issue by krkhan** [ryanoasis/vim-devicons#324]
#### Required Info

- [x] I have searched the [issues](https://github.com/ryanoasis/vim-devicons/issues) for my issue and found nothing related and/or helpful
- [x] I have read or at least glanced at the [FAQ](https://github.com/ryanoasis/vim-devicons/wiki/FAQ-&-Troubleshooting)
- [x] I have read or at least glanced at the [Wiki](https://github.com/ryanoasis/vim-devicons/wiki)
- [x] `VimDevIcons` is the last loaded vim plugin?
- [x] What **version** of vim are you using? Repros with both vim8 and neovim 0.4
- [x] What **version** of vim-devicons are you using? 58e57b6eeb407dba7ff6e607342c08c32b6fd398
- [x] Are you using vim from the terminal or a GUI vim? Terminal (Alacritty)
- [x] Are you using Mac, Linux or Windows? Linux

#### Optional Info

- [x] Provide **screenshots** where appropriate

![image](https://user-images.githubusercontent.com/697967/80270712-56902300-866f-11ea-8cdf-a71e485188d7.png)

Pretty consistent and happens with both vim and nvim. When I navigate _up_ a directory in NerdTREE the devicon is missing for the directory I was just in. Fo.r example, in the screenshot I just navigated up from the `teams` directory.

**Comment by aminnairi** [comment]
This also happen when I add a new file/folder. As a quickfix, I have found the reload action (`Shift-r`) in NERDTree to display the icons back (for the files/folder that had missing icons).

**Comment by krkhan** [comment]
@aminnairi Good idea. Since I was already using chdir mode 3 (which changes CWD when going up/down a tree), the following helped me automatically refresh:

```
let g:NERDTreeChDirMode = 3

function NERDRefresh ()
  if &filetype == "nerdtree"
      NERDTreeRefreshRoot
  endif
endfunction

call NERDRefresh()
```

**Comment by kazukazuinaina** [comment]
Could you check this issue?

https://github.com/ryanoasis/vim-devicons/issues/224
